qemu/tests/cfg/nic_acpi_index.cfg

```diff
@@ -0,0 +1,28 @@
+- nic_acpi_index:
+    virt_test_type = qemu
+    type = nic_acpi_index
+    only Linux
+    no q35
+    clone_master = yes
+    master_images_clone = image1
+    remove_image_image1 = yes
+    nic_hotplug_count=1
+    nic_model = virtio-net-pci
+    required_qemu = [6.0.0,)
+    nic_name_number = 2
+    kernel_extra_params_remove = biosdevname=0 net.ifnames=0
+    kernel_extra_params_serial_login = yes
+    variants:
+        - with_acpi:
+            vhost = on
+            nic_extra_params += ",acpi-index=1,addr=0x5"
+            type = nic_acpi_index_boot
+            nic_name_number = 1
+        - hotplug_same_slot:
+            nics = ""
+            nic_extra_params_hotplug_nic1 = ",acpi-index=2,addr=0x5"
```

Comment:
```suggestion
    kernel_extra_params_remove = biosdevname=0 net.ifnames=0
    kernel_extra_params_serial_login = yes
    variants:
        - with_acpi:
            vhost = on
            nic_name_number = 1
            type = nic_acpi_index_boot
        - hotplug_same_slot:
            nics = ""
            nic_name_number = 2
    nic_extra_params += ",acpi-index=${nic_name_number},addr=0x5"
```

Reply:
Thanks for your suggestion. I have changed it.